db/migration/V5__initial_data.sql

```diff
@@ -0,0 +1,9 @@
+insert into users(id, name, email, phone, role)
```

Comment:
In spring boot you have the idea of schema.sql or data.sql just in case you want to take that into consideration with the naming

Reply:
done